src/test_api.cpp

```diff
@@ -421,35 +424,35 @@ void test_merge_partial_dyn_mat() {
     pms_err[0] = pm2;
     pms_err[1] = pm3;
 
-    err = merge_partial_to_matrix(pms_err, 3, &obs);
+    err = merge_partial_to_mmap_matrix(pms_err, 3, NULL, &obs);
     ASSERT(err == incomplete_stripe_set);
 
     pm2->stripe_start = 2;
     pm2->stripe_stop = 6;
-    err = merge_partial_to_matrix(pms_err, 3, &obs);
+    err = merge_partial_to_mmap_matrix(pms_err, 3, NULL, &obs);
     ASSERT(err == stripes_overlap);
 
     pm2->stripe_start = 3;
     pm2->sample_ids[2][0] = 'X';
-    err = merge_partial_to_matrix(pms_err, 3, &obs);
+    err = merge_partial_to_mmap_matrix(pms_err, 3, NULL, &obs);
     ASSERT(err == sample_id_consistency);
 
     pm2->sample_ids[2][0] = 'C';
     pm3->n_samples = 2;
-    err = merge_partial_to_matrix(pms_err, 3, &obs);
+    err = merge_partial_to_mmap_matrix(pms_err, 3, NULL, &obs);
     ASSERT(err == partials_mismatch);
 
     pm3->n_samples = 6;
     pm3->stripe_total = 12;
-    err = merge_partial_to_matrix(pms_err, 3, &obs);
+    err = merge_partial_to_mmap_matrix(pms_err, 3, NULL, &obs);
     ASSERT(err == partials_mismatch);
     
     /*
      * Disable for now... not dealing properly with is_upper_triangle == false
```

Comment:
is this bad?

Reply:
we were using internal interfaces before.
Switch to the public interfaces to be able to use the shared library.